sacro-app/src/create-window.js

```diff
@@ -28,6 +29,7 @@ const createWindow = async () => {
   const result = await dialog.showOpenDialog({
     title: "Choose ACRO outputs json file",
     properties: ["openFile"],
+    defaultPath: os.homedir(),
```

Comment:
FWIW, there's also [an API in electron](https://www.electronjs.org/docs/latest/api/app#appgetpathname) to do the same thing. It might be a bit nicer, although I don't think it really makes any difference in practice.